zeppelin-server/src/main/java/org/apache/zeppelin/server/ZeppelinServer.java

```diff
@@ -295,9 +296,10 @@ private static Server setupJettyServer(ZeppelinConfiguration conf) {
     final Server server = new Server(threadPool);
     ServerConnector connector;
 
+    HttpConfiguration httpConfig = new HttpConfiguration();
+    httpConfig.addCustomizer(new ForwardedRequestCustomizer());
```

Comment:
Is it needed when it doesn't use proxy ?

Reply:
No it's not needed, when you doesn't use a proxy. Without a Proxy (e.g. SSL-Offload-Proxy)[`ForwardedRequestCustomizer`](https://www.eclipse.org/jetty/javadoc/9.4.8.v20171121/org/eclipse/jetty/server/ForwardedRequestCustomizer.html) has no impact, because the HTTP-Headers (`X-Forwarded-For`, `X-Forwarded-Proto` and so on) are missing.

Reply:
Thanks for the explanation. PR LGTM